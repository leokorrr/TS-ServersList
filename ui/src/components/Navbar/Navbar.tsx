import React from 'react'
import Logo from '../Logo/Logo'

const Navbar: React.FC = () => <nav className="navbar"><Logo title="Recruitment task"/></nav>

export default Navbar