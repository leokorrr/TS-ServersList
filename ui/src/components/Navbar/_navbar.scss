.navbar {
    height: 7.3rem;
    background-color: $color-nav-bg;
    display: flex;
    align-items: center;
}