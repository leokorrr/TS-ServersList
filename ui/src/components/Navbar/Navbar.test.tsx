import React from 'react'
import Navbar from './Navbar'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })
it("Navbar rendered without crashing", () => {
    shallow(<Navbar />);
})
it("should render initial layout", () => {
    const component = shallow(<Navbar />);
    expect(component.getElements()).toMatchSnapshot();
})