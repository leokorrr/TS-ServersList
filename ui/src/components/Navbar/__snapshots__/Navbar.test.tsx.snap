// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render initial layout 1`] = `
Array [
  <nav
    className="navbar"
  >
    <Logo
      title="Recruitment task"
    />
  </nav>,
]
`;
